### ISSUE TITLE
[catbuffer/parser] 3 cat files could not be parsed

### ISSUE THREAD

**Issue by xembook** [symbol/symbol#276]
Errors occur during parsing in 3 cat files.

- address_resolution_statement.cats 
- mosaic_resolution_statement.cats
- restriction_account_types.cats

```
$python3 -m catparser --schema ../schemas/symbol/statements/address_resolution_statement.cats  --include ../schemas/symbol -o output/address_resolution_statement.yaml


processing ../schemas/symbol/statements/address_resolution_statement.cats...
processing ../schemas/symbol/statements/receipt_source.cats...
[ERRORS DETECTED AT STAGE PRE EXPANSION]
 + [AddressResolutionEntry::{ resolved_value }] reference to unknown type "Address"
 + [AddressResolutionStatement::{ unresolved }] reference to unknown type "UnresolvedAddress"
```

```
$python3 -m catparser --schema ../schemas/symbol/statements/mosaic_resolution_statement.cats  --include ../schemas/symbol -o output/mosaic_resolution_statement.yaml


processing ../schemas/symbol/statements/mosaic_resolution_statement.cats...
processing ../schemas/symbol/statements/receipt_source.cats...
[ERRORS DETECTED AT STAGE PRE EXPANSION]
 + [MosaicResolutionEntry::{ resolved_value }] reference to unknown type "MosaicId"
 + [MosaicResolutionStatement::{ unresolved }] reference to unknown type "UnresolvedMosaicId"
 ```

```
$python3 -m catparser --schema ../schemas/symbol/state/restriction_account_types.cats  --include ../schemas/symbol -o output/restriction_account_types.yaml


processing ../schemas/symbol/state/restriction_account_types.cats...
processing ../schemas/symbol/entity.cats...
processing ../schemas/symbol/types.cats...
processing ../schemas/symbol/transaction_type.cats...
[ERRORS DETECTED AT STAGE PRE EXPANSION]
 + [AccountRestrictionsInfo::{ restriction_flags }] reference to unknown type "AccountRestrictionFlags"
 + [AccountRestrictionsInfo::{ address_restrictions }] field value "ADDRESS" is not a valid numeric value
 + [AccountRestrictionsInfo::{ mosaic_id_restrictions }] field value "MOSAIC_ID" is not a valid numeric value
 + [AccountRestrictionsInfo::{ transaction_type_restrictions }] field value "TRANSACTION_TYPE" is not a valid numeric value
```

**Comment by gimre-xymcity** [comment]
released as a part of client/catapult/v1.0.3.4

**Comment by cryptoBeliever** [comment]
Tested and output yaml files are now generated correctly.
